transforms/images/roi-relabel-plugin/src/polus/plugins/transforms/images/roi_relabel/methods/__init__.py

```diff
@@ -44,15 +63,15 @@ def _relabel_tile(
 
 
 def relabel(
```

Comment:
I don't think it's necessary now, but we may want to consider doing threading inside this function to complement the process pool executor outside of this function. Right now, it looks like this would run my faster on one hundred 1024x1024 images than it would on a single 10240x10240 image. Maybe that difference isn't significant considering the complexity of the operations we are performing.

Reply:
I added threading. Not from `preadator` though because I'm still working on fixing that.